### PR TITLE
Add fixtures to test edge cases in the web console

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@smarterclayton

File: test/assets/fixtures/k8s-lonely-pod.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "kind": "List",
```
Comment (liggitt) [inline test/assets/fixtures/k8s-lonely-pod.json]: is this right? I would have expected `PodList`

Comment (jwforres) [inline test/assets/fixtures/k8s-lonely-pod.json]: yeah its a generic list of things, each item specifies its type.  I ran all of these with `osc create -f whatever.json`

**Comment by liggitt** [comment]
Since the overview page wires up lots of interrelated objects, seems like we should set up groups of related fixtures (pods, replication controllers, services, builds, etc), with a way to select an entire group at a time.

**Comment by smarterclayton** [comment]
I don't think these are asset specific - can you move them to `test/fixtures/app-scenarios` or similar?

File: test/assets/fixtures/k8s-lonely-pod.json (hunk 1 above)
Comment (smarterclayton) [inline test/assets/fixtures/k8s-lonely-pod.json]: Make sure these get tested by examples_test.go

----- Original Message -----

**Comment by jwforres** [comment]
Moved

**Comment by smarterclayton** [comment]
LGTM [merge]
